features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/accessibility/AccessibilitySnapshotManager.kt

```diff
@@ -10,5 +10,5 @@ import com.datadog.tools.annotation.NoOpImplementation
 
 @NoOpImplementation
 internal interface AccessibilitySnapshotManager {
-    fun latestSnapshot(): AccessibilityInfo
+    fun latestSnapshot(): AccessibilityInfo?
```

Comment:
by the usage at the call site maybe it is better to call this as `getIfChanged`, but not a blocker for me, just a suggestion.